tensornetwork/__init__.py

```diff
@@ -1,6 +1,9 @@
 from __future__ import absolute_import
+
 from tensornetwork.network import TensorNetwork
-from tensornetwork.network_components import Node, Edge, CopyNode
+from tensornetwork.network_components import Node, Edge, CopyNode, BaseNode
+#pylint: disable=line-too-long
+from tensornetwork.network_components import conj, transpose, contract, contract_copy_node, contract_between, outer_product, outer_product_final_nodes, split_node, split_node_qr, split_node_rq, split_node_full_svd, reachable, contract_parallel, check_connected, check_correct, connect, flatten_edges, flatten_all_edges, flatten_edges_between, get_all_nondangling, get_all_nodes, get_all_edges, get_parallel_edges, get_shared_edges, _contract_trace
```

Comment:
Can you put these in a different file other than `network_components`? Perhaps `network_operations` or something similar?

Reply:
OK, can try, this will cause some cyclic impoprt issues with network_operations.connect and disconnect. I think I will have to keep those in network_components.py

Reply:
That's fine by me.

Reply:
added new file network_operations.py